.github/CONTRIBUTING.md

```diff
@@ -44,6 +53,8 @@ Note: All repos use [conventional commits](https://www.conventionalcommits.org)
 
 ## 5. Submit a Pull Request
 
+‼️ **Temporarily, please wait to open a Pull Request until it's ready for review. We need to optimize build time.**
```

Comment:
this line can get removed, we have fixed this issue :)

Reply:
✅